main.js

```diff
@@ -281,9 +281,9 @@ tvBrandName(inventory);
 // Opdracht 4a: Maak een herbruikbare functie die een string genereert voor de naam van één tv en deze teruggeeft in het format [merk] [type] - [naam] zoals Philips 43PUS6504/12 - 4K TV of NIKKEI NH3216SMART - HD smart-TV
 //---------------------------------------------------------------//
 function brandTypeName(tv){
-   return tv.brand + " " + tv.type + " - " + tv.name;
- }
- console.log(brandTypeName(inventory[0]));
+  return tv.brand + " " + tv.type + " - " + tv.name;
```

Comment:
Wellicht dat je dit de volgende keer zo kan doen
```suggestion
  return `${tv.brand} ${tv.type} - ${tv.name}`;
```
Dit vind ik altijd voor mezelf wat leesbaarder, wellicht jij dus ook 😉